[2025-03-21T21:00:08.309+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T21:00:08.318+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_twitter.fetch_twitter_data_task scheduled__2025-03-21T15:00:00+00:00 [queued]>
[2025-03-21T21:00:08.325+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_twitter.fetch_twitter_data_task scheduled__2025-03-21T15:00:00+00:00 [queued]>
[2025-03-21T21:00:08.325+0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-03-21T21:00:08.342+0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_twitter_data_task> on 2025-03-21 15:00:00+00:00
[2025-03-21T21:00:08.348+0500] {standard_task_runner.py:72} INFO - Started process 104286 to run task
[2025-03-21T21:00:08.352+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_twitter', 'fetch_twitter_data_task', 'scheduled__2025-03-21T15:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/fetch_twitter.py', '--cfg-path', '/tmp/tmp0ien0duz']
[2025-03-21T21:00:08.353+0500] {standard_task_runner.py:105} INFO - Job 29: Subtask fetch_twitter_data_task
[2025-03-21T21:00:08.390+0500] {task_command.py:467} INFO - Running <TaskInstance: fetch_twitter.fetch_twitter_data_task scheduled__2025-03-21T15:00:00+00:00 [running]> on host ubuntu
[2025-03-21T21:00:08.459+0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_twitter' AIRFLOW_CTX_TASK_ID='fetch_twitter_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T15:00:00+00:00'
[2025-03-21T21:00:08.460+0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-21T21:00:08.460+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-21T21:00:08.461+0500] {logging_mixin.py:190} INFO - Current task name:fetch_twitter_data_task state:running start_date:2025-03-21 16:00:08.319153+00:00
[2025-03-21T21:00:08.461+0500] {logging_mixin.py:190} INFO - Dag name:fetch_twitter and current dag run status:running
[2025-03-21T21:00:08.461+0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T21:00:08.905+0500] {logging_mixin.py:190} INFO - Error fetching tweets: 403 Forbidden
453 - You currently have access to a subset of X API V2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.x.com/en/portal/product
[2025-03-21T21:00:08.905+0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T21:00:08.909+0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T21:00:08.910+0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fetch_twitter, task_id=fetch_twitter_data_task, run_id=scheduled__2025-03-21T15:00:00+00:00, execution_date=20250321T150000, start_date=20250321T160008, end_date=20250321T160008
[2025-03-21T21:00:08.930+0500] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-21T21:00:08.930+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-21T21:00:08.931+0500] {logging_mixin.py:190} INFO - Dag name:fetch_twitter queued_at:2025-03-21 16:00:01.204151+00:00
[2025-03-21T21:00:08.931+0500] {logging_mixin.py:190} INFO - Task hostname:ubuntu operator:PythonOperator
[2025-03-21T21:00:08.966+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T21:00:08.978+0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T21:00:08.978+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
